README.md

```diff
@@ -1,2 +1,8 @@
 # bakedpdf
 Bake a PDF
+
+npm install
```

Comment:
Can you wrap these commands with backticks? That way it is easier to see at a glance that these are commands that need to be run